tutorials/arvo/claz.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "claz"
+weight = 4
+template = "doc.html"
++++
+
+This document outlines some advanced techniques for interacting with Azimuth contracts from Urbit itself. It's strongly discouraged to do this unless you're very confident as Azimuth transactions will cost Etherium to complete. However, until you upload a signed transaction to the block chain no transaction will take place so it's safe to explore these tools. It will also be very helpful to have at least a rudimentary understanding of Hoon in order to read and modify parts of the code.
+
+First you will want to boot a fakezod. You can do this with your live running ship but at this time this will need you to modify the code so it's best to do this on a ship you can make mistakes on. If you don't know how to boot a fakezod see this guide for instructions.
+
+Be sure to have mounted your home desk so you can easily edit files on it.
+The first edit that will probably need to be made is in [`app/claz.hoon`](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/claz.hoon#L14). The listed url will need to be changed to 
+
+https://mainnet.infura.io/v3/2599df54929b47099bda360958d75aaf
+
+or another valid endpoint.
+
+The second change that will potentially be needed is to modify the [gas limit](https://github.com/urbit/urbit/blob/master/pkg/arvo/app/claz.hoon#L179). This number here is the maximum number of gas units that will be used. In future this will be configurable but for now you may need to change this manually. For a guide to what to set it to you can use the [constants in bridge](https://github.com/urbit/bridge/blob/master/src/lib/constants.js#L23) as a guide based on what action you are going to be preforming. You will need at least the maximum gas units multiplied by the gwei you're going to set later in your etherium account as it may cost that much though is likely to be far less.
```

Comment:
You don't _need_ to change it, but it has implications for transaction submission and max fee.

Reply:
> You will need at least the maximum gas units multiplied by the gwei you're going to set later

I realize this is immediately followed by "in your Ethereum account", but the way the given calculation directly follows example numbers confused me into thinking you were saying you need to edit the limit to max * gwei or w/e.

Probably better to phrase as "The maximum cost of your transaction will be ... . You'll need at least that in your account." or similar.